src/main/webapp/style.css

```diff
@@ -5,3 +5,33 @@ h3 {
 .srp-container {
   display: flex;
 }
+
+#auth {
+  width: 47px;
+  height: 15px;
+  color: white;
+  background: #4d90fe;
+  border: 1px #3079ED solid;
+  padding: 5px 10px;
+  border-radius: 2px;
+  font: 400 13.3333px Arial;
+  text-align: center;
+  text-decoration: none;
+  font-size: 9pt;
+  outline: none;
```

Comment:
Why? This looks like it's disabling the focus ring that keyboard navigation users depend on.